test/helpers/getPaginationLink.js

```diff
@@ -0,0 +1,20 @@
+import chai from 'chai';
+import getPaginationLink from 'helpers/getPaginationLink';
+
+const { assert } = chai;
```

Comment:
Should we use 

```
import { assert } from 'chai';
```

instead of 
```
import chai from 'chai';
const { assert } = chai;
```
? Same here https://github.com/iris-dni/iris-frontend/pull/67/files#diff-65b5fd2ae5e0bcfe4b9d044f60ee077eR5